code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Toxins.dm

```diff
@@ -158,6 +158,52 @@
 	T.assume_gas(GAS_PHORON, amount, T20C)
 	remove_self(amount, holder)
 
+/singleton/reagent/toxin/phoron/kois
+	name = "Phoron"
+	description = "Phoron in its liquid form. Twice as potent when breathed in. Contains biological traces."
+	fallback_specific_heat = 1 //contaminated phoron acts more like k'ois than phoron
+	var/kois_type = 1
```

Comment:
this needs to be a define like KOIS_TYPE_NORMAL

Reply:
var/kois_type = KOIS_TYPE_NORMAL?